jmix-reports/reports/src/main/java/io/jmix/reports/yarg/formatters/impl/XlsxFormatter.java

```diff
@@ -857,14 +913,27 @@ protected Row createNewRow(Worksheet resultSheet) {
         return newRow;
     }
 
+    private Integer computeColumnIndex(String cellName) {
```

Comment:
I suggest you to cover this logic with unit test.

Reply:
added